api/v1/condition_test.go

```diff
@@ -404,11 +404,7 @@ func TestChangeMaptoDynamoMap(t *testing.T) {
 				"age":     map[string]interface{}{"N": "20"},
 				"value":   map[string]interface{}{"N": "10"},
 				"array": map[string]interface{}{
-					"L": []map[string]interface{}{
```

Comment:
So now we are mapping array to string set instead of list. This is a change in behavior and may break existing applications just like this test case.